bbs/admin/unix/install.sh

```diff
@@ -326,14 +331,22 @@ then
   fi
 else
   say "User ${WWIV_USER} not found; creating it."
-  useradd -g ${WWIV_GROUP} -c "WWIV BBS Service Account" -d ${WWIV_DIR} \
-	  -s /sbin/nologin -m ${WWIV_USER} >> ${LOGFILE} 2>&1
+  if [[ $OSTYPE =~ "SunOS" ]]
+  then
```

Comment:
That would be a better choice, but we also have to deal with variations in different linux distros.  /sbin/nologin actually isn't used in all distros.  uname -s isn't enough to tell them apart.